.github/stale_example.yml

```diff
@@ -0,0 +1,69 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+ daysUntilStale: 50
+
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 10
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - "Priority: P0"
+  - "Priority: P1"
+  - "Priority: P2"
+  - "Progress: dev in progress"
+  - "Progress: PR in progress"
+  - "Progress: done"
+  - "B2B: GraphQL"   
+  - "Progress: PR Created"
+  - "PAP"
+  - "Project: Login as Customer"
+  - "Project: GraphQL"
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: false
```

Comment:
Should we use this config instead of label Project: GraphQL?

Reply:
Agree it sounds reasonable but
1. its not clear what exactly means "Project" in this context. I assume all projects on this page  https://github.com/magento/magento2/projects?query=is%3Aopen
 I do not see Project: GraphQL here. Seems we treat them only by labels

2. for   "Project: Login as Customer". Now the current implementation of the board app, automatically adds each new issue to the mainboard magento2/issue. And automatically adds it if any action will be triggered(assignee, pr created etc.). I did not check how the bot will manage the case if the Issue card is added to the 2 projects (more then 1)

My proposition leave it as is(manage by labels only) for now
After we have completed all planned migration and unification for tasks for bords/sub-project apps, we can adjust it

Reply:
ok